easybuild/easyconfigs/p/Perl/Perl-5.32.1-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,1798 @@
+        'checksums': ['dbf7c827984951fb248907f940fd8f19f2696bc5545c0a15287e0fbe56a52308'],
+    }),
+    ('if', '0.0608', {
+        'source_tmpl': 'if-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XS/XSAWYERX'],
+        'checksums': ['37206e10919c4d99273020008a3581bf0947d364e859b8966521c3145b4b3700'],
+    }),
+    ('Test', '1.26', {
+        'source_tmpl': 'Test-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/J/JE/JESSE'],
+        'checksums': ['f7701bd28e05e7f82fe9a181bbab38f53fa6aeae48d2a810da74d1b981d4f392'],
+    }),
+    ('ExtUtils::Constant', '0.25', {
+        'source_tmpl': 'ExtUtils-Constant-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/N/NW/NWCLARK'],
+        'checksums': ['6933d0e963b62281ef7561068e6aecac8c4ac2b476b2bba09ab0b90fbac9d757'],
+    }),
+    ('ExtUtils::CBuilder', '0.280236', {
+        'source_tmpl': 'ExtUtils-CBuilder-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/A/AM/AMBS'],
+        'checksums': ['abc21827eb8a513171bf7fdecefce9945132cb76db945036518291f607b1491f'],
+    }),
+    ('Carp::Heavy', '1.50', {
+        'source_tmpl': 'Carp-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XS/XSAWYERX'],
+        'checksums': ['f5273b4e1a6d51b22996c48cb3a3cbc72fd456c4038f5c20b127e2d4bcbcebd9'],
+    }),
+    ('IO::Handle', '1.45', {
+        'source_tmpl': 'IO-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TO/TODDR'],
+        'checksums': ['3b104b3a86ad2b7ed760e0ef4a35a6e16a67f3f4c24d2f661654dfbe5272e100'],
+    }),
+    ('IO::Seekable', '1.45', {
+        'source_tmpl': 'IO-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TO/TODDR'],
+        'checksums': ['3b104b3a86ad2b7ed760e0ef4a35a6e16a67f3f4c24d2f661654dfbe5272e100'],
+    }),
+    ('Pod::Simple', '3.42', {
+        'source_tmpl': 'Pod-Simple-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/K/KH/KHW'],
+        'checksums': ['a9fceb2e0318e3786525e6bf205e3e143f0cf3622740819cab5f058e657e8ac5'],
+    }),
+    ('IO::File', '1.45', {
+        'source_tmpl': 'IO-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TO/TODDR'],
+        'checksums': ['3b104b3a86ad2b7ed760e0ef4a35a6e16a67f3f4c24d2f661654dfbe5272e100'],
+    }),
+    ('IO::Select', '1.45', {
+        'source_tmpl': 'IO-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TO/TODDR'],
+        'checksums': ['3b104b3a86ad2b7ed760e0ef4a35a6e16a67f3f4c24d2f661654dfbe5272e100'],
+    }),
+    ('IO::Socket', '1.45', {
+        'source_tmpl': 'IO-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TO/TODDR'],
+        'checksums': ['3b104b3a86ad2b7ed760e0ef4a35a6e16a67f3f4c24d2f661654dfbe5272e100'],
+    }),
```

Comment:
Everything from `IO-%(version)s.tar.gz` is part of Perl itself i think so we shouldn't need to include it. https://metacpan.org/release/perl
(i skimmed through the rest, and I don't think there was anything else)

Reply:
Thanks @Micket! I'll have a look into #12627 and remove IO-related stuff.

Reply:
Added #12627 and removed IO-related stuff.

Reply:
Please see https://github.com/easybuilders/easybuild-easyconfigs/pull/12611#discussion_r616570368 we shouldn't include it after all